src/pages/public/account/LoginPage.vue

```diff
@@ -27,16 +32,26 @@ export default {
         FormCard,
         PasswordInput,
     },
+    metaInfo: {
+        title: "Login",
+        titleTemplate: null
+    },
     data() {
         return {
             email: '',
             password: '',
-            isEmailError: false
+            isLoginError: false
         }
     },
     methods: {
-        validateEmail() {
-            this.isEmailError = !(/^\S+@\S+\.\S+$/.test(this.email))
+        async formSubmit() {
+            try {
+               await this.$store.dispatch("user/login", { email: this.email, password: this.password})
```

Comment:
I propose to move `{ email: this.email, password: this.password}` to a separate variable, to make the code a little bit cleaner, what do you think? :)
For instance:
```
const loginData  = {
  email: this.email,
  password: this.password
};
await this.$store.dispatch("user/login", loginData);
```

Reply:
Good call! cleaner = better